### PR TITLE
chore: add `repository.directory` in `package.json` for all packages

### PR DESCRIPTION
**Pull request by iamandrewluca**
This is the new way to mark a package in a monorepo.
See [repository.directory](https://docs.npmjs.com/files/package.json#repository) property in `package.json`.
In a future release of `npmjs.com` this will allow to link directly to a package in a monorepo.

https://docs.npmjs.com/files/package.json#repository
https://github.com/npm/rfcs/blob/latest/implemented/0010-monorepo-subdirectory-declaration.md

Related https://github.com/material-components/material-components-web-react/pull/975
Related https://github.com/material-components/material-components-web-components/pull/302

### CODE REVIEW

**Comment by iamandrewluca** [comment]
I signed it!

**Comment by iamandrewluca** [comment]
@abhiomkar this is the new way to mark a package in a monorepo see [repository.directory](https://docs.npmjs.com/files/package.json#repository) property in `package.json`. In a future release of `npmjs.org` this will allow to link directly to package in monorepo.

File: packages/material-components-web/package.json [hunk 1]
```diff
@@ -13,7 +13,8 @@
   "types": "dist/material-components-web.d.ts",
   "repository": {
     "type": "git",
-    "url": "https://github.com/material-components/material-components-web"
+    "url": "https://github.com/material-components/material-components-web",
```
Comment (abhiomkar) [inline packages/material-components-web/package.json]: `.git` is missing.

**Comment by abhiomkar** [comment]
Thanks @iamandrewluca for explaining!

**Pull request merged**